components/toggl/actions/create-project/create-project.mjs

```diff
@@ -0,0 +1,138 @@
+import toggl from "../../toggl.app.mjs";
+
+export default {
+    name: "Create a new Project",
+    version: "0.0.1",
+    description: "Create project for given workspace. [See docs here.](https://developers.track.toggl.com/docs/api/projects#post-workspaceprojects)",
+    type: "action",
+    key: "toggle-create-new-project",
+    props: {
+        toggl,
+        name: {
+            type: "string",
+            label: "Name",
+            description: 'unique for client and workspace'
+        },
+        workspace_id: {
+            type: "integer",
+            label: "Workspace ID",
+            description: "Numeric ID of the workspace"
+        },
+        client_id: {
+            type: 'integer',
+            label: 'client ID',
+            optional: true,
+        },
+        client_name: {
+            type: 'string',
+            label: 'client Name',
+            optional: true,
+        },
+        active: {
+            type: 'boolean',
+            label: 'Active',
+            optional: true,
+            description: ' Whether the project is archived or not',
+        },
+        is_private: {
+            type: 'boolean',
+            label: 'Is Private',
+            optional: true,
+            description: 'Whether project is accessible for only project users or for all workspace users',
+        },
+        template: {
+            type: 'boolean',
+            label: 'Template',
+            optional: true,
+            description: 'Whether the project can be used as a template',
+        },
+        template_id: {
+            type: 'integer',
+            label: 'Template ID',
+            description: "Id of the template project used on current project's creation",
+            optional: true,
+        },
+        billable: {
+            type: 'boolean',
+            label: 'Billable',
+            description: "Whether the project is billable or no (available only for pro workspaces)",
+        },
+        auto_estimates: {
+            type: 'boolean',
+            label: 'Auto Estimates',
+            optional: true,
+            description: "whether the estimated hours are automatically calculated based on task estimations or manually fixed based on the value of 'estimated_hours' (premium functionality) ",
+        },
+        estimated_hours: {
+            type: 'boolean',
+            label: 'Estimated Hours',
+            optional: true,
+            description: "if auto_estimates is true then the sum of task estimations is returned, otherwise user inserted hours (premium functionality)"
```

Comment:
```suggestion
            description: "If `Auto Estimates` is true then the sum of task estimations is returned, otherwise user inserted hours (premium functionality)"
```